adapters/parent/pom.xml

```diff
@@ -248,5 +258,131 @@
         </dependency>
       </dependencies>
     </profile>
+    
+    <profile>
+      <id>read-ports</id>         
+      <activation>
+        <property>
+          <name>read-ports</name>
+        </property>
+      </activation>
+      <properties>
+        <docker.dir>${project.basedir}/../../tests/target/docker</docker.dir>
```

Comment:
Can we define the location to be a static path outside of the source tree, e.g. something below the user's home directory, e.g. something like ${user.home}/hono/config ?
That way we would not need to traverse within the source tree, which is kind of prone to break when module layout is changing ...

Reply:
It's arguable if it is a good idea for a maven project to write outside the project home. After all the user may mind  if it's home directory (for instance) is filled with unexpected data.
So, I'd propose to leave it like this. Later it could be thought how to improve that.